Charter assessment dev task from Alexander Treshchev

treshev opened the pull request:
Good morning, 

My implementation in this pull request.
Unfortunately, I was not able to push it like a feature branch due to expected "Access Denied" error.

dnellis74 commented on README.md:
```diff
@@ -40,4 +40,11 @@ AWS or Heroku, or whatever you're comfortable with will be an "A+"
\ No newline at end of file
+questions section, push it, and we will update the file with answers. 
+
+### Blake Yarbrough Progress
```
Thanks for updating the readme.md!

dnellis74 commented on src/main/java/com/charter/enterprise/motd/MotdController.java:
```diff
@@ -1,16 +1,32 @@
 @RestController
 public class MotdController {
 
+    //bad idea
```
Your comment is spot on, but this is indeed an okay solution, give the amount of time we expected you to spend.

dnellis74 commented on src/test/java/com/charter/enterprise/motd/MotdControllerTest.java:
```diff
@@ -26,4 +29,15 @@ public void getMotd() throws Exception {
     }
+
+    @Test
+    public void postMotd() throws Exception {
```
Great job adding the test too.

david-nellis-charter commented:
Thanks Blake.   I've saved your branch in a private repo.

treshev committed:
added 2 methods:

1) updateMessageFromPutRequest
recommended method. Just update the message of the day by put request.

2) updateMessageFromGetRequest
Not recommended, but might be used in case if the user does not have any tools like a curl or telnet
but have to be able to update message.
In this case, the message might be updated using the link in any browser
for example "/updateMessage?newMessageOfTheDay=Hello World!"

treshev committed:
Changes:
 1. Fix the getMotdTest method

2. Add 2 new tests for the GET and PUT requests which change the message of the day

3. Annotation @DirtiesContext was added to clear context after each test.
   Due to the fact that the  message now stored in a local variable in the controller
   which has the singleton scope, the context should be cleared after each test which change the message.
   Otherwise, the getMotdTest might be failed if it runs after any other.

treshev committed:
Update README.md

update README as was mentioned in the task
